homeassistant/components/sensor/speedtest.py

```diff
@@ -143,9 +146,13 @@ def update(self, now):
 
         _LOGGER.info('Executing speedtest')
         try:
+            args = [sys.executable, speedtest_cli.__file__, '--simple']
+            if self._server_id:
+                args = args.append('--server')
+                args = args.apped(self._server_id)
```

Comment:
Yup thanks, wrote it very quick after a friend suggested it to me. Will fix
shortly

On Aug 28, 2016 1:18 AM, "Martin Hjelmare" <notifications@github.com> wrote:

> In homeassistant/components/sensor/speedtest.py
> <https://github.com/home-assistant/home-assistant/pull/3008#discussion_r76527496>
> :
>
> > @@ -143,9 +146,13 @@ def update(self, now):
> >
> >          _LOGGER.info('Executing speedtest')
> >          try:
> > +            args = [sys.executable, speedtest_cli.__file__, '--simple']
> > +            if self._server_id:
> > +                args = args.append('--server')
> > +                args = args.apped(self._server_id)
>
> Typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3008/files/18178c4c22823ddf4be7e4e2d82594fd2ea23603#r76527496>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZf2Vqk3JNH7NifdlAuPGEyy8_wAjks5qkTZKgaJpZM4JuxlV>
> .
>